src/containers/confirm-audio-to-waudio/desktop/ConfirmAudioToWAudioModal.module.css

```diff
@@ -0,0 +1,80 @@
+.modalTitle {
+  font-size: var(--font-2xl);
+}
+
+.titleEmoji {
+  margin-left: 16px;
+}
+
+.modalBody {
+  max-width: 480px;
+  width: 100%;
+}
+
+.body {
+  padding: 32px 46px 24px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.bodyText {
+  font-weight: var(--font-demi-bold);
+  font-size: 18px;
+  line-height: 26px;
+  text-align: center;
+  color: var(--neutral);
+}
+
+.header {
+  font-weight: var(--font-demi-bold);
+  font-size: 22px;
+  line-height: 26px;
+  text-align: center;
+  color: var(--secondary);
+
+  margin-bottom: 16px;
+}
+
+.moreInfo {
+  font-weight: var(--font-demi-bold);
+  font-size: var(--font-m);
+  line-height: 30px;
+  text-align: center;
+  color: var(--neutral-light-4);
+  cursor: pointer;
+  margin: 8px auto 32px;
+}
+
+.moreInfo:hover,
+.moreInfo:active {
+  color: var(--neutral);
+  text-decoration: underline !important;
```

Comment:
yes because we have this global style: https://github.com/AudiusProject/audius-client/blob/5276a3d836612fe87d9e641dc0abf667edf1a8d9/src/index.css#L51

Reply:
> yes because we have this global style:
> 
> https://github.com/AudiusProject/audius-client/blob/5276a3d836612fe87d9e641dc0abf667edf1a8d9/src/index.css#L51

that's awful.